api/src/chat/services/context-var.service.ts

```diff
@@ -30,7 +30,7 @@ export class ContextVarService extends BaseService<ContextVar> {
     block: Block | BlockFull,
   ): Promise<Record<string, ContextVar>> {
     const vars = await this.find({
-      name: { $in: block.capture_vars.map(({ context_var }) => context_var) },
+      name: { $in: block.capture_vars?.map(({ context_var }) => context_var) },
```

Comment:
Doesn't the capture_vars defaults to an empty array ?

Reply:
Yes the capture_vars is defaults empty array
But the schema type of the field don't reflect the value (capture_vars can have an undefined type)
![image](https://github.com/user-attachments/assets/54aefdba-772a-452a-9fea-e60d13f85d77)